Sources/Vapor/Sessions/SessionsMiddleware.swift

```diff
@@ -7,9 +7,7 @@
 /// on the outgoing response with the session's unique identifier. This cookie must be
 /// returned on the next request to restore the session.
 ///
-///     var middlewareConfig = MiddlewareConfig()
-///     middlewareConfig.use(SessionsMiddleware.self)
-///     services.register(middlewareConfig)
+///     app.middleware.use(SessionsMiddleware.self)
```

Comment:
As per [the docs](https://docs.vapor.codes/4.0/sessions/#configuration), this should be
```suggestion
///     app.middleware.use(app.sessions.middleware)
```